pkg/client/client.go

```diff
@@ -37,66 +36,139 @@ import (
 
 const (
 	defaultEndpointPort = "9440"
-	ProviderName        = "nutanix"
-	configPath          = "/etc/nutanix/config"
-	endpointKey         = "prismCentral"
-	capxNamespaceKey    = "POD_NAMESPACE"
+
+	capxNamespaceKey = "POD_NAMESPACE"
+)
+
+var (
+	// use a var to override in tests
+	configPath = "/etc/nutanix/config/prismCentral"
```

Comment:
Given we are using `readManagerNutanixPrismEndpointFromFile` in tests, can we can make this a const?

Reply:
Yep forgot in the refactor, thanks!